src/makeCAF.C

```diff
@@ -373,19 +437,22 @@ void loop(CAF &caf,
     // reset (the default constructor initializes its variables)
     caf.setToBS();
 
-    double pot = par().runInfo().POTPerSpill() * 1e13;
-    if (std::isnan(caf.pot))
-      caf.pot = 0;
-    caf.pot += pot;
-    caf.sr.beam.pulsepot = pot;
-    caf.sr.beam.ismc = true;  // when we have data, we won't be able to use this "POT from config" approach anyway
 
+    std::vector<double> trigger_times;
     // hand off to the correct reco filler(s).
     for (const auto & fillerTrigPair : groupedTriggers[ii])
     {
       cafmaker::LOG_S("loop()").INFO() << "Global trigger idx : " << ii << ", reco filler: '" << fillerTrigPair.first->GetName() << "', reco trigger eventID: " << fillerTrigPair.second.evtID << "\n";
       fillerTrigPair.first->FillRecoBranches(fillerTrigPair.second, caf.sr, par, &truthMatcher);
+      trigger_times.push_back(fillerTrigPair.second.triggerTime_s + 1e-9*fillerTrigPair.second.triggerTime_ns);
     }
+    //Fill POT
+    double pot = getPOT(par, trigger_times, ii);
+    if (std::isnan(caf.pot))
+      caf.pot = 0;
+    caf.pot += pot;
+    caf.sr.beam.pulsepot = pot;
+    caf.sr.beam.ismc = true; //should be handled in a smarter way for data and MC
```

Comment:
This isn't right any more.  Shouldn't you be using the presence of a spill file or something like that to determine it?  i.e.
```suggestion
    caf.sr.beam.ismc = par().cafmaker().POTFile.hasValue();  // fixme: when we have proper IFDB interface, should use the same mechanism as however we decide when to use that
```